### PR TITLE
cere: upgrade and expand telepad network

### PR DESCRIPTION
**Pull request by warriorstar-orion**
## What Does This PR Do
This PR upgrades the quantum pad in two ways: spawning them with efficiency equal to T4 components, and giving their areas important APCs. It then expands the network, adding stops on the Medbay and Service asteroids, and connecting them to the corners on the opposite sides of the station.

Note that the T4 components don't actually exist in the pads, to prevent them being used for R&D or in other machines. Instead, the pads' efficiency is hard-coded.

## Why It's Good For The Game
Starting the pads upgraded makes sense considering their importance in navigating the station, and increasing their APC cell charge by default ensures they're more reliable as the station goes to shit. Adding more stops in the network makes it much, much easier for people to get around.
## Images of changes

Demo video

https://github.com/ParadiseSS13/Paradise/assets/59303604/4b25fefe-d6f2-4402-9769-0c6a858fde04

### Screenshots

<details><summary>SDMM</summary>

2024_04_11__19_46_43__paradise dme  cerestation dmm  - StrongDMM
2024_04_11__19_46_54__paradise dme  cerestation dmm  - StrongDMM
2024_04_11__19_47_00__paradise dme  cerestation dmm  - StrongDMM
2024_04_11__19_47_05__paradise dme  cerestation dmm  - StrongDMM
2024_04_11__19_47_12__paradise dme  cerestation dmm  - StrongDMM
2024_04_11__20_25_18__paradise dme  cerestation dmm  - StrongDMM

</details>

## Testing
Spawned in, teleported around.

## Changelog
:cl:
add: Farragus: the quantum pad network has been upgraded and expanded. The upgraded parts cannot be taken out of the telepads.
fix: The description for quadratic capacitors was missing a word, which has been fixed.
/:cl:

### CODE REVIEW

**Comment by meow20** [comment]
Finally first use of the service direction tag lol

**Comment by spacegeese** [comment]
thank you for making fartgas more tolerable

File: code/game/machinery/quantum_pad.dm [hunk 1]
```diff
@@ -45,6 +65,12 @@
 	. = ..()
 	linked_pad = locate(preset_target)
 
+/obj/machinery/quantumpad/cere/PopulateParts()
+	// No parts in Cere telepads, just hardcode the effiencies
```
Comment (Henri215) [inline code/game/machinery/quantum_pad.dm:69]:
```suggestion
	// No parts in Cere telepads, just hardcode the efficiencies
```

File: code/game/objects/items/weapons/stock_parts.dm [hunk 2]
```diff
@@ -220,7 +220,7 @@
 
 /obj/item/stock_parts/capacitor/quadratic
 	name = "quadratic capacitor"
-	desc = "An capacity capacitor used in the construction of a variety of devices."
+	desc = "An ultra-high capacity capacitor used in the construction of a variety of devices."
```
Comment (DGamerL) [inline code/game/objects/items/weapons/stock_parts.dm:223]: ... how did nobody notice this before

**Review by 1080pCat — COMMENTED** [review]
Why have you put a vender in the kitchen?
image

Im not the biggest fan of how cramped this tunnel is. there is alot of free _space_ to the right of the area, and the area designation a little odd in the two pointed parts. the lower one could be swapped for the maints area, and the top one could be the unexplored
image

You've got a stray disposal pipe here
image

Stray decal here
image

Supply pipes aren't connected to the vent pump
image

Stray disposal pipe here
image

Light on the shutter
image

Theres no pipes connected to the other scrubbers/vents here
image

You've added another display here, but seemingly forgot to remove the original one
image

Theres also one here, probably dont need 2 in the same hall in view of eachother
image

On the displays, they're not bumped. but it seems to be like that on all of farragus? so D:

Geology fans love this window.
image

Why have you abbreviated service to `SRUC`? `SERV` would do better.
image

This door would be better as a `/public/glass` (the white ones) rather than the grey one
image

Could you repipe the pipes to be like this?
image

As a final request, could you add fire locks to all of the doors? there are fire alarms, but no.. locks? Im pretty sure you didn't forget them, but since you're redoing them anyways.

**Comment by warriorstar-orion** [comment]
> Why have you put a vender in the kitchen? image

Originally the pads opened straight into the bar, this was an artifact of that. Removed.

> 
> Im not the biggest fan of how cramped this tunnel is. there is alot of free _space_ to the right of the area, and the area designation a little odd in the two pointed parts. the lower one could be swapped for the maints area, and the top one could be the unexplored image

I've expanded this a bit and fixed the two areas you mentioned, PTAL. Regarding whether rock walls in maints count as the maint area or unexplored mining, Cere does a little bit of both. Port maintenance includes the rock walls in the maint area, but med maints only includes constructed walls. I went with the latter but there should probably be a standard around this.

> 
> You've got a stray disposal pipe here image
> 

Fixed.

> Stray decal here image

Fixed.


> 
> Supply pipes aren't connected to the vent pump image
> 

Fixed.

> Stray disposal pipe here image
> 

Fixed.

> Light on the shutter image

Fixed. 


> Theres no pipes connected to the other scrubbers/vents here image
> 
> You've added another display here, but seemingly forgot to remove the original one image
> 

Fixed.

> Theres also one here, probably dont need 2 in the same hall in view of eachother image
> 

Fixed.

> On the displays, they're not bumped. but it seems to be like that on all of farragus? so D:
 
I'll probably fix all these when I make directional helpers for them.

> Geology fans love this window. image

Fixed.

> Why have you abbreviated service to `SRUC`? `SERV` would do better. image

That's the same V used in the sign for "EVAC", and was meant to be an abbreviation of SeRViCe. I've changed it to SERV and darkened the colors a tiny bit so that it looks more like a V and less like a U.

> This door would be better as a `/public/glass` (the white ones) rather than the grey one image

Fixed.

> Could you repipe the pipes to be like this? image
> 

Fixed.

> As a final request, could you add fire locks to all of the doors? there are fire alarms, but no.. locks? Im pretty sure you didn't forget them, but since you're redoing them anyways.

Fixed.

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**